### DISCUSSION TITLE
Tradução de Artigos

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#1400] [Discussão]
Pessoal o que vocês acham de tradução de artigos, mesmo colocando a fonte do autor?

Eu quero consolidar um pouco meu aprendizado, e já sou tradutor na Mozilla, e isso tem me ajudado muito com Javascript. 
Mas sou pessimo em escrever artigos do zero eu mesmo, e tenho uma coleção muito grande de artigos bons que aprendi muito e me ajudam muito no desenvolvimento todos os dias.

E sei que pra muitos iniciantes ia ajudar bastante, porque muitas pessoas não sabem o que procurar ou não sabem inglês.

Vocês acham uma boa ideia? Ou acham que é uma má pratica, porque parece que passa uma autoria de algo que não foi você que fez? Por exemplo, eu tenho uma coleção de mais de 300 artigos sobre varios temas aqui no meu pocket, desde React , a node, a TDD, Js etc.. e muitos temas não tem em portugues.

e pra consolidar meu conhecimento e tb ajudar pensei em começar a traduzir um por dia.

Mas tenho receio que seja visto como cópia apenas, e não é esse o intuito.

**Comment by rafaelnogueira1** [comment]
Cara já vi em alguns sites, bem conhecidos até, que tinham arquivos traduzidos. Acredito que entrando em contato com autor e dando referencia não seja um problema.

E se tu for tocar esse projeto mesmo e quiser colaborador eu to dentro cara.

**Comment by rubenmarcus** [comment]
@rafaelnogueira1  a maioria dos artigos eu vejo no medium ou no scotch.io, e outros blogs.. acho uma ideia boa, eu queria um dia ter um site que fosse um site de vagas voltado a front-end com blogs,palestras e tb mentoria pro pessoal novo que está entrando na area.

**Comment by felipefialho** [comment]
Mano, todas as vezes que fiz tradução pedi autorização para o autor original do texto.

**Comment by HmmerHead** [comment]
Não vejo como uma cópia. Acredito que contribui muito para a expansão do conhecimento de quem for ler. Nem sempre um cara Sênior, com conhecimento na língua, vai procurar sobre o assunto. 

Muitas vezes, pessoas com menos conhecimento da língua, precisam saber sobre o assunto, para um trabalho de faculdade, ou ainda no novo emprego por exemplo, ou até mesmo por curiosidade.

Temos muitos outros cenários, mas o importante é poder difundir esse assunto.

Se for o seu desejo, você deve investir nisso sim. E deve principalmente procurar o dono e pedir a sua autorização. Estou bem certo que ele vai gostar, já que vai ajudar a espalhar as suas idéias para outros países.

**Comment by rubenmarcus** [comment]
Entendi, acho que o lance da autorização é legal mesmo, mas tem devs que só tem medium, ou a rede social é só o twitter, e varios nem respondem a comentários. Ou você não encontra e-mails

**Comment by rubenmarcus** [comment]
Vou dar exemplo, estou começando a fazer um projeto Angular em TDD, e só vi 1 artigo em portugues até agora. Em ingles encontrei mais de 30. Minha vontade era de traduzir pelo menos uns 15. Porque sei que tem gente que não tem a mesma facilidade de encontrar o material. Mas se for pedir autorização para os 30 autores, muito provavelmente vários não vao responder.

**Comment by vitor-mendrone-deel** [comment]
Traduções são sempre uma boa por tornarem o conhecimento universal/inclusivo

Porém é fundamental a autorização do autor, se o mesmo não responder ou não autorizar é melhor partir pra outro artigo, independente da qualidade ou da importância do artigo a decisão deve vir sempre do autor.

**Comment by ghost** [comment]
Complementando a resposta do @Mendrone, caso não tenha autorização do autor, ou não tenha uma resposta sequer, talvez seja uma boa escrever seu próprio artigo, sintetizando o que aprendeu durante o projetos e as leituras, sempre tomando cuidado com plágios, etc. Caso algum desses artigos tenha sido uma referência, linka corretamente no fim do seu artigo, para dar os devidos créditos.

**Comment by ninetails** [comment]
Só tou jogando uma idéia: pq não um repositório, github mesmo, onde a galera reune e traduz colaborativamente? quem achar um post interessante e conseguir autorização poderia jogar lá (numa issue) e a galera se organizaria igual num [fansub](https://pt.wikipedia.org/wiki/Fansub), cada um (ou mais de um colaborador se juntaria e) traduz(iriam) um(a parte) de um artigo e o deploy publicaria no medium do grupo, creditando o autor original e o(s) tradutor(es).

Sei lá, pode rolar conflito de interesses lá na frente mas só to jogando uma idéia...

**Comment by rafaelnogueira1** [comment]
Ia fazer a mesma sugestão que o @GuimaraesJean , tenta o contato pelo Twitter por exemplo, se não tiver uma resposta junta uns 5 artigos e cria um novo a partir daí. O bom disso é que ainda vai juntar com alguma experiência que você também teve no projeto. Não monetizando em cima do trabalho de outro, dá pra ser bem negociado isso aí.

**Comment by tarciozemel** [comment]
Acho uma excelente ideia. Ajuda tanto você próprio, que vai aprender mais, quanto à comunidade de devs.

Fazemos isso há bastante tempo no [desenvolvimentoparaweb.com](http://desenvolvimentoparaweb.com).

Mas é legal/"ético" citar os originais, sim. Dá uma olhada lá nos artigos do blog; a primeira coisa é sempre a citação da fonte do original.
